<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1196</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeWidget" name="treeWidget">
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLineEdit" name="searchWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>50</height>
             </size>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>NEXT</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTextEdit" name="textEdit"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1196</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClear"/>
    <addaction name="actionExpand_All"/>
    <addaction name="actionCollapse_All"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionExpand_All">
   <property name="text">
    <string>Expand All</string>
   </property>
  </action>
  <action name="actionCollapse_All">
   <property name="text">
    <string>Collapse All</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
